samples/net/wpan_serial/src/main.c

```diff
@@ -508,9 +508,9 @@ static bool init_ieee802154(void)
 {
 	SYS_LOG_INF("Initialize ieee802.15.4");
 
-	ieee802154_dev = device_get_binding(CONFIG_IEEE802154_CC2520_DRV_NAME);
+	ieee802154_dev = device_get_binding(CONFIG_NET_APP_IEEE802154_DEV_NAME);
```

Comment:
Following https://github.com/zephyrproject-rtos/zephyr/pull/9391, these should be renamed: CONFIG_NET_APP_* => CONFIG_NET_CONFIG_* (not all NET_APP_* of course, only those related to config).